src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/mappers/EfilingFilingPackageMapper.java

```diff
@@ -12,28 +12,29 @@
 
 @Mapper
 public interface EfilingFilingPackageMapper {
-    @Mapping(target = "courtFileNo", source = "filingPackage.court.fileNumber")
-    @Mapping(target = "entUserId", source = "clientId")
+    @Mapping(target = "courtFileNo", source = "submission.filingPackage.court.fileNumber")
+    @Mapping(target = "entUserId", source = "submission.accountDetails.clientId")
     @Mapping(target = "existingCourtFileYn", constant = "false")
     @Mapping(target = "processingCompleteYn", constant = "false")
     @Mapping(target = "feeExemptYn", constant = "false")
     @Mapping(target = "cfcsaYn", constant = "false")
     @Mapping(target = "automatedProcessYn", constant = "false")
     @Mapping(target = "notificationRequiredYn", constant = "false")
-    @Mapping(target = "ldcxCourtClassCd", source = "filingPackage.court.courtClass")
-    @Mapping(target = "ldcxCourtLevelCd", source = "filingPackage.court.level")
-    @Mapping(target = "submittedToAgenId", source = "filingPackage.court.agencyId")
-    @Mapping(target = "submittedByAccountId", source = "accountId")
-    @Mapping(target = "submittedByClientId", source = "clientId")
-    @Mapping(target = "ldcxCourtDivisionCd", source = "filingPackage.court.division")
-    EfilingFilingPackage toEfilingFilingPackage(Submission submission);
+    @Mapping(target = "ldcxCourtClassCd", source = "submission.filingPackage.court.courtClass")
+    @Mapping(target = "ldcxCourtLevelCd", source = "submission.filingPackage.court.level")
+    @Mapping(target = "submittedToAgenId", source = "submission.filingPackage.court.agencyId")
```

Comment:
should this be:
```suggestion
    @Mapping(target = "submittedToAgencyId", source = "submission.filingPackage.court.agencyId")
```

Reply:
just two extra characters but makes it much easier to understand what that var is

Reply:
unfortunatly it's comming from the cso object, nothing we can do about it...

Reply:
Yep those are not under our control.